Lib9c/BlockPolicy.cs

```diff
@@ -160,6 +159,24 @@ private InvalidBlockException ValidateBlock(Block<NCAction> block)
                 return null;
             }
 
+            // As a temporary approach to prevent selfish mining (again), we add a new rule
+            // disallowing blocks with less than 10 transactions.  This rule is applied since
+            // 1,870,000th block.  (Note that as of Jul 8, 2021, there are about 1,860,000+ blocks.)
+            // This rule is not applied to blocks (with proofs) made by authorized miners.
+            if (block.Transactions.Count < 10 &&
+                (IgnoreHardcodedIndicesForBackwardCompatibility || block.Index >= 1_870_000) &&
```

Comment:
We may need to change this value by the timeline. 😅

Reply:
👌🏼 Let me know when the index number needs to be changed.

Reply:
@longfin As we've already passed by [1,870,000th block](https://explorer.libplanet.io/9c-main/block/?0f413ff7fabe6aa150b5989979ae5c0d46bbdc261a893a5f9243ec217ace658b), we need to change the target block index.